sdk/dwallet-mpc-wasm/src/lib.rs

```diff
@@ -19,6 +20,23 @@ pub fn create_dkg_centralized_output(
         .map_err(|e| JsError::new(&e.to_string()))
 }
 
+/// Derives a Secp256k1 class groups keypair from a given seed.
+///
+/// The class groups key that is being used to encrypt a Secp256k1 keypair should be different from
+/// the encryption key used to encrypt a Ristretto keypair, due to cryptographic reasons.
+/// This function derives a class groups keypair to encrypt a Secp256k1 secret from the given seed.
+#[wasm_bindgen]
+pub fn generate_secp_cg_keypair_from_seed(seed: &[u8]) -> Result<JsValue, JsError> {
+    let fixed_size_seed: [u8; 32] = seed.try_into().expect("seed must be 32 bytes long");
+    let (public_key, private_key) =
+        generate_secp_cg_keypair_from_seed_internal(
+            fixed_size_seed,
+        )
+            .map_err(to_js_err)?;
+    Ok(serde_wasm_bindgen::to_value(&(public_key, private_key))?)
+}
```

Comment:
```suggestion
#[wasm_bindgen]
pub fn generate_secp_cg_keypair_from_seed(seed: &[u8]) -> Result<JsValue, JsError> {
    let (public_key, private_key) =
        generate_secp_cg_keypair_from_seed_internal(
            seed.try_into().expect("seed must be 32 bytes long"),
        )
            .map_err(to_js_err)?;
    Ok(serde_wasm_bindgen::to_value(&(public_key, private_key))?)
}
```
If it doesn't work, then just remove the magic number.